index.html

```diff
@@ -1439,16 +1468,14 @@
 André Mazzone,
 Nikie Monteleone,
 Michael Nickelsky,
-Anton Palmqvist,
```

Comment:
It looks like Anton and Nicolas were accidentally removed in this change!

Reply:
Will fix

Reply:
Done in 8821f630914bb6eb0f952970479c1f6d3067b690 (also added some others).